authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/UiResourceImpl.kt

```diff
@@ -172,6 +172,7 @@ class UiResourceImpl : UiResource {
     @Transactional
     override fun deactivateParticipantUser(userId: String): IdResponse {
         authUtils.requiresRole(Roles.UserRoles.PARTICIPANT_ADMIN)
+        authUtils.requiresRegistrationStatus(userId, UserRegistrationStatus.ACTIVE)
```

Comment:
Does it make sense to restrict this to `ACTIVE`? What if I want to deactivate a user that has been invited by someone?

Also, not clear who the function refers to, maybe name it `requiresTargetRegistrationStatus` to be more consistent?

Reply:
This makes sense, because otherwise INVITED/PENDING users can be deactivated and then activated, leading to an inconsistens state.